tests/marqo_test.py

```diff
@@ -124,7 +126,18 @@ def setUpClass(cls) -> None:
             print(f"MARQO SERVER VERSION -> {marqo_server_version}")
             print(f"PY-MARQO SUPPORTED VERSION -> {py_marqo_support_version}")
 
-
+    @classmethod
+    def tearDownClass(cls) -> None:
+        """Delete commonly used test indexes after all tests are run
+        """
+        client = marqo.Client(**cls.client_settings)
+        commonly_used_ix_name = 'my-test-index-1'
```

Comment:
I feel like we should have a single place where we define the indexes a class will be using. MarqoTestCase can have some generic ones, and subclasses can override this by "appending" more.